dataflow/manual/content.tex

```diff
@@ -913,19 +912,14 @@ \subsection{Formal Definition of the Control-Flow Graph}
 \end{workinprogress}
 
 \begin{workinprogress}
-``Could be desugared'' on ``StringConcatenateAssignment'' and ``Conversion
+``Could be desugared'' on ``Conversion
 nodes'' was confusing.  What is the design rationale for
 desugaring in the Dataflow Framework?   Discuss that.  Here, at least
 forward-reference to \autoref{sec:desugaring}, if that's relevant. \\
 More generally, for any cases that will be discussed in the text, add a
 forward reference to the section with the discussion.
 \end{workinprogress}
 
-\begin{workinprogress}
```

Comment:
Please add something about the desugaring in sec:desugaring.

Reply:
fixed.